display.py

```diff
@@ -0,0 +1,53 @@
+from comms import Comms, Message
+from datetime import datetime
+from datetime import time
+import json
+import multiprocessing as mp
+import os
+import random
+import sys
+import threading
+import time
+
+
+class display_base:
+    def __init__(self, fileN):
+        self.comms = Comms()
+        m_list = json.loads(open(fileN).read())
+        self.state = dict()
+        port_number = 4000
+        
+        for item in m_list['mlist']:
```

Comment:
I would separate this out into a function like "init_subs" or something. This could go in the service abstract class.

Reply:
Speaking of which, should each service have their own config? Not all of them should start with the same connections, right?